### PR TITLE
Ignore build_x86_64/release dirs in format scripts

### PR DESCRIPTION
**Pull request by cg2121**
### Description
The scripts would fail if these directories were present.

### Motivation and Context
Found when formatting plugin.

### How Has This Been Tested?
Ran script to see if it didn't fail

### Types of changes
- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: .github/scripts/check-cmake.sh [hunk 1]
```diff
@@ -36,7 +36,8 @@ if ! type cmake-format 2> /dev/null ; then
 fi
 
 find . -type d \( \
-    -path ./\*build -o \
+    -path ./\*build\* -o \
+    -path ./release \
```
Comment (RytoEX) [inline .github/scripts/check-cmake.sh:40]: Shouldn't there be another -o after release?

File: .github/scripts/check-format.sh [hunk 2]
```diff
@@ -42,7 +42,8 @@ else
 fi
 
 find . -type d \( \
-    -path ./\*build -o \
+    -path ./\*build\* -o \
+    -path ./release \
```
Comment (RytoEX) [inline .github/scripts/check-format.sh:46]: Shouldn't there be another -o after release?

**Review by RytoEX — CHANGES_REQUESTED** [review]

File: .github/scripts/check-cmake.sh (hunk 1 above)
Comment (cg2121) [reply .github/scripts/check-cmake.sh]: Fixed

File: .github/scripts/check-format.sh (hunk 2 above)
Comment (cg2121) [reply .github/scripts/check-format.sh]: Fixed

**Review by RytoEX — APPROVED** [review]
This looks correct visually. @PatTheMav ?

**Review by PatTheMav — APPROVED** [review]

**Pull request merged**